apps/prairielearn/elements/pl-sketch/pl-sketch.mustache

```diff
@@ -0,0 +1,50 @@
+<div class="pl-sketch" id="pl-sketch-{{uuid}}">
+    <div class="pl-sketch-controls" style="width: {{width}}px">
+        <div class="pl-sketch-control-group">
+            <div class="pl-sketch-control" id="undo"><div class="fa fa-undo fa-2x"></div></div>
+            <div class="pl-sketch-control" id="redo"><div class="fa fa-redo fa-2x"></div></div>
+            <div class="pl-sketch-control" id="pen"><div class="fa fa-pen fa-2x"></div></div>
+            <div class="pl-sketch-control" id="eraser"><div class="fa fa-eraser fa-2x"></div></div>
```

Comment:
Although I acknowledge that this element isn't going to be particularly accessible to keyboard users since there's no straightforward way for them to draw, we still should not be using a `<div>` as a clickable thing. A `<button>` is absolutely the way to go.